src/test/java/com/google/sps/TimingPropertiesTest.java

```diff
@@ -44,63 +44,67 @@ public void tearDown() {
  
     @Test
     //Test where user has null timestamp
-    public void getTimestampProperty_nullUserTimeStamp() {
+    public void getQuizTimestampProperty_nullUserTimeStamp() {
         QuizTimingPropertiesUtils timing_properties_test = new QuizTimingPropertiesUtils();
         Entity user = new Entity("user");
         user.setProperty("quiz_timestamp", null);
+        user.setProperty("userID", "abc");
  
         DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
         datastore.put(user);
  
-        Long actual = timing_properties_test.getTimestampProperty("user", datastore); 
+        Long actual = timing_properties_test.getQuizTimestampProperty("user", "userID", (user.getProperty("userID")).toString(), datastore); 
         Assert.assertEquals(null, actual);
     }
  
     @Test
     //Test where String parameter is empty
-    public void getTimestampProperty_emptyEntityString() {
+    public void getQuizTimestampProperty_emptyEntityString() {
         QuizTimingPropertiesUtils timing_properties_test = new QuizTimingPropertiesUtils();
- 
         DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
  
-        Long actual = timing_properties_test.getTimestampProperty("", datastore);
+        Long actual = timing_properties_test.getQuizTimestampProperty("", "userID", "abc", datastore);
         Assert.assertEquals(null, actual);
     }
  
     @Test
     //Test where datastore object is null
-    public void getTimestampProperty_noDatastore() {
+    public void getQuizTimestampProperty_noDatastore() {
         QuizTimingPropertiesUtils timing_properties_test = new QuizTimingPropertiesUtils();
  
         Entity user = new Entity("user");
-        Long actual = timing_properties_test.getTimestampProperty("user", null);
+        user.setProperty("userID", "abc");
+ 
+        Long actual = timing_properties_test.getQuizTimestampProperty("user", "userID", (user.getProperty("userID")).toString(), null);
         Assert.assertEquals(null, actual);
```

Comment:
This might be one of those where you want to check for a null pointer exception instead of returning null

Reply:
so I have try catch statements in the helper functions which is why I don't have them in the test. That is why the tests are still returning null. I can move the try catch statements over but I just thought it would be cleaner to keep it in the helper functions

Reply:
Emma is right, it would be fine to not try/catch at all and just let the NPE happen (and test to make sure it happens) but we can save that for a future PR dedicated to cleaning up things like this.